package.json

```
@@ -4,7 +4,8 @@
   "description": "",
   "scripts": {
     "start": "nodemon -e ts --exec ts-node server",
-    "ganache": "nodemon ganache"
+    "ganache": "nodemon ganache",
+    "test": "truffle test"
```

Might as well add `truffle migrate` to `package.json`?
```suggestion
    "migrate": "truffle migrate",
    "test": "truffle test"
```

---

Done ✅